### PR TITLE
mkosi: Update to latest

### PR DESCRIPTION
**Pull request by daandemeyer**
The Bootable= option was removed and mkosi installs less packages by default now, so let's adapt our configs to those changes.

### CODE REVIEW

**Review by yuwata — APPROVED** [review]

File: mkosi.conf.d/20-debian.conf [hunk 1]
```diff
@@ -10,6 +10,7 @@ Release=testing
 Packages=
         btrfs-progs
         cryptsetup-bin
+        dbus
```
Comment (bluca) [inline mkosi.conf.d/20-debian.conf:13]: Let's use dbus-broker here and in ubuntu

File: mkosi.conf.d/20-debian.conf [hunk 2]
```diff
@@ -10,6 +10,7 @@ Release=testing
 Packages=
         btrfs-progs
         cryptsetup-bin
+        dbus-broker
```
Comment (bluca) [inline mkosi.conf.d/20-debian.conf]: sorry forgot to mention, you'll also need to pull in default-dbus-session-bus

**Review by bluca — APPROVED** [review]

**Comment by bluca** [comment]
something weird is happening, looks like the broker is not starting on debian (but it is starting on ubuntu), wtf

**Comment by bluca** [comment]
@DaanDeMeyer there is something weird going on in debian so also install `dbus` there for now please, I'll try to figure it out separately

**Pull request merged**